### DISCUSSION TITLE
SNDH files by !Cube

### DISCUSSION THREAD

**Discussion by rnjacobs** [frno7/psgplay#12] [Q&A] [answered]
I was hoping to listen to sndh files composed by !cube - http://cube.gfxile.net/t/index.php?page=journal&journal=YM2149F - but none of them work in psgplay. Am I doing something wrong? Is he doing something unusual? Should they work?

Thank you!

**Comment by frno7** [comment, marked as answer]
I suppose it should, so I registered issue #13 for it!
